examples/vm_scheduling/reinforcement_learning/ac/agent/ac_net.py

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import numpy as np
```

Comment:
It is OK in here as the code for example.

But according to the rules, since numpy and torch are both 3rd party package, it should be:

```
import numpy as np
import torch
....
``` 

(no blank lines between)